parsers/yamlparser.go

```diff
@@ -177,16 +177,17 @@ func (yaml *YAML) GetProject() Project {
 	return yaml.Application
 }
 
-func convertSinglePackageName(packageName string) string {
-	if len(packageName) != 0 {
-		packageNameEnv := wskenv.GetEnvVar(packageName)
-		if str, ok := packageNameEnv.(string); ok {
+
+func convertSingleName(theName string) string {
```

Comment:
Can you move this function to the package wskenv please? (i.e., wskenv/environment.go )?